server/src/main/kotlin/org/javacs/kt/index/SymbolIndex.kt

```diff
@@ -177,11 +177,11 @@ class SymbolIndex {
         fqName.toString().length <= MAX_FQNAME_LENGTH
             && fqName.shortName().toString().length <= MAX_SHORT_NAME_LENGTH
 
-    fun query(prefix: String, receiverType: FqName? = null, limit: Int = 20): List<Symbol> = transaction(db) {
+    fun query(prefix: String, receiverType: FqName? = null, limit: Int = 20, wildcardMatcher: String = "%"): List<Symbol> = transaction(db) {
```

Comment:
Perhaps a more generic name like `suffix` instead of `wildcardMatcher` (since that's only the default use case)?